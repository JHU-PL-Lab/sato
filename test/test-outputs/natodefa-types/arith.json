[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "x" ],
          "err_match_val": "false",
          "err_match_expected": "Integer",
          "err_match_actual": "Boolean"
        }
      ],
      [
        "Error_match",
        {
          "err_match_aliases": [ "y" ],
          "err_match_val": "false",
          "err_match_expected": "Integer",
          "err_match_actual": "Boolean"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "x + y",
    "err_steps": 112
  }
]