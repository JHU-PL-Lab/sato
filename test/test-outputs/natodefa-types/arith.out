Type errors for:
- Input sequence : []
- Found at expr  : x + y
--------------------
* Value    : x = false
* Expected : Integer
* Actual   : Boolean
--------------------
* Value    : y = false
* Expected : Integer
* Actual   : Boolean
2 errors found. No further control flows exist.
